metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -927,6 +927,28 @@ class MetalsLanguageServer(
         textDocumentSyncOptions.setSave(new SaveOptions(true))
         textDocumentSyncOptions.setOpenClose(true)
 
+        val scalaFilesPattern = new FileOperationPattern("**/*.scala")
+        scalaFilesPattern.setMatches(FileOperationPatternKind.File)
+        val scFilesPattern = new FileOperationPattern("**/*.sc")
```

Comment:
Not sure if we need to receive notifications for script files

Reply:
We don't need them since the scripts cannot contain package definition